akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowFutureFlowSpec.scala

```diff
@@ -256,32 +256,32 @@ class FlowFutureFlowSpec extends StreamSpec {
 
       }
 
-      "handle closed downstream when flow future is late completed" in assertAllStagesStopped {
+      "handle closed downstream when flow future is completed after downstream cancel" in assertAllStagesStopped {
         val prFlow = Promise[Flow[Int, Int, Future[collection.immutable.Seq[Int]]]]()
-        val (fSeq1, fSeq2) = src10()
+        val (fNestedFlowMatVal, fSinkCompletion) = src10()
           .viaMat {
             Flow.futureFlow(prFlow.future)
           }(Keep.right)
           .mapMaterializedValue(_.flatten)
-          .take(0)
-          .toMat(Sink.seq)(Keep.both)
+          .take(0) // cancel asap
+          .toMat(Sink.ignore)(Keep.both)
           .withAttributes(attributes)
           .run()
 
-        if (delayDownstreanCancellation) {
-          fSeq1.value should be(empty)
```

Comment:
If the Future[Seq[]] from Sink.seq was completed here or not is not deterministic. Also, the test kind of depends on the cancellation or else it isn't testing "late complete"